### PR TITLE
Dev/user dash backend

### PR DESCRIPTION
**Pull request by hochangyong**
### **Partially done with user dashboard (need you guys' help)**

**Done**
- Able to read and display users
- Only display up to 9 users in one page
- Fix the responsive design that unorganized previously
- Able to bring to the user profile when click the specific user

**Need Help**
- Don't display the user itself
- Open to next page and back to previous page

**Sorry for finish it so late and partially done only

### CODE REVIEW

**Comment by EHeng17** [comment]
Will endless scrolling be easier to implement to counter the page issue faced by chang yong?

**Comment by JacksonTai** [comment]
It can be, but we definitely cannot load all questions at once when the user loads the page. We can try to load more questions when the user scrolls to the bottom of the page.

**Comment by EHeng17** [comment]
I found 1 or 2 external resources about using php and ajax to load content. Let me give it a shot and see if I can make it work.
I will let u guys know if I need help

**Comment by EHeng17** [comment]
Good news, I have already created that function, scrolling will load additional data
I set it to load 12 users, 9 takes up a little of the screen only

Im just having a hard time combining it into controller and model. I will try to get it done by tonight

**Comment by JacksonTai** [comment]
Nice! Feel free to reach out if you're stuck with the controller and model.

**Comment by JacksonTai** [comment]
The chat page shouldn't be including backend code yet. It should be the same as what the master branch has.

**Comment by EHeng17** [comment]
Chang Yong, do u know what u have to remove according to what Jackson said?

**Comment by hochangyong** [comment]
> Chang Yong, do u know what u have to remove according to what Jackson said? If not, I'll remove it for u

you mean the chat page ?

**Comment by hochangyong** [comment]
okay, i will remove it and push again

**Comment by EHeng17** [comment]
I have made the necessary changes to the chat's css, frontend and backend according to master

**Review by EHeng17 — APPROVED** [review]

File: bash.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Stack trace:
+Frame        Function    Args
+000FFFFB440  0018006286E (0018028EC28, 0018026DE3E, 00000000000, 000FFFFA340)
+000FFFFB440  0018004846A (00000000000, 00000000000, 00000000000, 001004E2F5F)
+000FFFFB440  001800484A2 (0018028ECD9, 000FFFFB2F8, 00000000000, 00000000000)
+000FFFFB440  001800CC6BE (00000000000, 00000000000, 00000000000, 00000000000)
+000FFFFB440  001800CC7E5 (000FFFFB450, 00000000000, 00000000000, 00000000000)
+00000000000  001800CDDA5 (000FFFFB450, 00000000000, 00000000000, 00000000000)
+End of stack trace
```
Comment (JacksonTai) [inline bash.exe.stackdump:9]: What's this?

Comment (EHeng17) [reply bash.exe.stackdump]: I have deleted the file, and re run the code, everything works fine

**Review by jyyyyylim — APPROVED** [review]

**Review by JacksonTai — APPROVED** [review]

**Pull request merged**